src/node/modules.h

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+#pragma once
+
+#include <msgpack/msgpack.hpp>
+#include <optional>
+#include <stdint.h>
+#include <string>
+#include <vector>
+
+namespace ccf
+{
+  struct Module
+  {
+    std::optional<std::string> js;
+    // std::optional<uint8_t> wasm;
```

Comment:
For now, I think we should just make this a required string member:

```suggestion
    std::string js;
```